src/backend/aspen/workflows/nextstrain_run/nextstrain_profile/ordering.tsv

```diff
@@ -1,1485 +1,23 @@
-division	Missouri
-division	Florida
-division	Georgia
-division	Alabama
-division	Arkansas
-division	Louisiana
-division	North Carolina
-division	South Carolina
-division	Virginia
-division	Rhode Island
-division	Connecticut
-division	New York
-division	New Jersey
-division	Massachusetts
-division	New Hampshire
-division	Vermont
-division	Pennsylvania
-division	Ohio
-division	Indiana
-division	Illinois
-division	Wisconsin
-division	Michigan
-division	Minnesota
-division	Iowa
-division	Nebraska
-division	Kansas
-division	Nevada
-division	Arizona
-division	Colorado
-division	New Mexico
-division	Utah
-division	Wyoming
-division	Idaho
-division	Montana
-division	California
-division	Grand Princess
-division	Grand Princess 2nd cruise
-division	Oregon
-division	Washington
-division	Hawaii
-division	Puerto Rico
-division	Virgin Islands
-division	Jamaica
-division	Alaska
-division	Canada
-division	Nova Scotia
-division	New Brunswick
-division	Quebec
-division	Ontario
-division	Manitoba
-division	Prince Edward Island
-division	Newfoundland and Labrador
-division	Saskatchewan
-division	British Columbia
-
-
-country	China
-country	Hong Kong
-country	Taiwan
-country	Philippines
-country	Japan
-country	South Korea
-country	Nepal
-country	India
-country	Bangladesh
-country	Sri Lanka
-country	Vietnam
-country	Cambodia
-country	Thailand
-country	Myanmar
-country	Malaysia
-country	Singapore
-country	Brunei
-country	Indonesia
-country	Guam
-country	Timor-Leste
-country	Australia
-country	New Zealand
-country	Saudi Arabia
-country	Qatar
-country	United Arab Emirates
-country	Kuwait
-country	Lebanon
-country	Oman
-country	Iran
-country	Israel
-country	Jordan
-country	Turkey
-country	Pakistan
-country	Kazakhstan
-country	Kyrgyzstan
-country	Georgia
-country	Russia
-country	Serbia
-country	Romania
-country	Europe
-country	Latvia
-country	Slovakia
-country	Slovenia
-country	Poland
-country	Belarus
-country	Estonia
-country	Lithuania
-country	Hungary
-country	Croatia
-country	France
-country	Iceland
-country	Norway
-country	Denmark
-country	Netherlands
-country	Belgium
-country	Luxembourg
-country	Germany
-country	Austria
-country	Switzerland
-country	Italy
-country	Spain
-country	Czech Republic
-country	United Kingdom
-country	Ireland
-country	Finland
-country	Sweden
-country	Portugal
-country	Greece
-country	Egypt
-country	Nigeria
-country	Kenya
-country	Ghana
-country	Senegal
-country	Gambia
-country	Algeria
-country	Uganda
-country	Democratic Republic of the Congo
-country	South Africa
-country	Chile
-country	Ecuador
-country	Peru
-country	Brazil
-country	Argentina
-country	Uruguay
-country	Colombia
-country	Panama
-country	Costa Rica
-country	Mexico
-country	Jamaica
-country	USA
-country	Canada
-
-recency	Older
-recency	One month ago
-recency	One week ago
-recency	3-7 days ago
-recency	1-2 days ago
-recency	New
+location	Cook County IL
+location	DuPage County
+location	Kane County IL
+location	Lake County IL
+location	McHenry County
+location	Will County IL
 
```

Comment:
Is an extra blank line going to cause problems?

Reply:
nope, I did a test run with this branch and all is fine :D